index.html

```diff
@@ -32,7 +32,9 @@ <h1 class="text-6xl text-color-primary-dark">Recipe Creator</h1>
           each ingredient with a comma and space.
         </p>
         <div class="flex flex-col items-center">
-        <textarea rows="1" cols="100" placeholder="Chicken, carrots, celery..." id="spoonIngredientInput" class="text-center w-3/4 text-4xl border-2-primary"></textarea>
+
+        <textarea rows="1" cols="100" placeholder="Chicken, carrots, celery..." id="spoonIngredientInput" class="text-center w-3/4 text-4xl"></textarea>
+        <select id="ingredientDropdown" class="text-center w-3/4"></select>
```

Comment:
Can we get the localStorage to save to the textarea instead of a separate dropdown?